### ISSUE TITLE
Publish on NPM?

### ISSUE THREAD

**Issue by moltar** [cuperman/cdk-appconfig#14]
Hey Jeff,

Any plans to publish this to NPM proper?

The GitHub registry isn't very ergonomic to install packages from. It requires a GitHub token, even when the packages are public.

**Comment by cuperman** [comment]
@moltar I can publish this on npm if you're interested in using it.

**Comment by moltar** [comment]
Yes, would love to try it. I think it's be nicest implementation I've see thus far 😁

**Comment by cuperman** [comment]
Ok, cool 😎 Let me set up a release pipeline for this and get it into npmjs.com.  Then I'll look at issue #15

**Comment by cuperman** [comment]
@moltar looks like AWS is going to support native high level AppConfig constructs going forward:
https://www.npmjs.com/package/@aws-cdk/aws-appconfig-alpha

so you may want to try using these instead.  you will get better long term support / maintenance.

**Comment by moltar** [comment]
Ah, awesome, had no idea this was already out. Thanks!!
